mythril/analysis/modules/dos.py

```diff
@@ -70,7 +71,7 @@ def _analyze_states(self, state: GlobalState) -> List[Issue]:
 
                 try:
                     self._jumpdest_count[transaction][target] += 1
-                    if self._jumpdest_count[transaction][target] == 4:
+                    if self._jumpdest_count[transaction][target] == JUMPDEST_LIMIT - 1:
```

Comment:
this would always go in for if conditions right? if JUMPDEST_LIMIT is 2

Reply:
no always but if there's more than 1 condition. So probably 4 is a more reasonable limit...